packages/mds-stream/kafka/helpers.ts

```diff
@@ -16,7 +16,10 @@
 
 export const getKafkaBrokers = () => {
   const {
-    env: { KAFKA_HOST = 'localhost:9092' }
+    env: { KAFKA_HOST }
   } = process
-  return [KAFKA_HOST]
+
+  if (KAFKA_HOST) return [KAFKA_HOST]
+
+  return null
```

Comment:
Should this throw (or at least log) to indicate that something is trying to use an unconfigured Kafka?

Reply:
A log message like "KAFKA_HOST is undefined - skipping initialization" would be useful.

Reply:
Added :)